org.cockpit_project.CockpitClient.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.cockpit_project.CockpitClient
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+default-branch: stable
+command: cockpit-client
+rename-desktop-file: cockpit-client.desktop
+rename-icon: cockpit-client
+modules:
+  - name: cockpit-client
+    buildsystem: autotools
+    config-opts:
+      - --enable-cockpit-client
+      - --disable-polkit
+      - --disable-ssh
+      - --disable-pcp
+      - --with-systemdunitdir=/invalid
+      - CPPFLAGS=-Itools/mock-build-env
+      - --with-admin-group=root
+      - --disable-doc
+    make-args:
+      - build-for-flatpak
+    install-rule: install-for-flatpak
+    post-install:
+      - install -Dt /app/share/metainfo ${FLATPAK_ID}.metainfo.xml
+
+    sources:
+      - type: archive
+        url: https://github.com/cockpit-project/cockpit/releases/download/258/cockpit-258.tar.xz
+        sha256: 5806a53218da6b4679eb6087251fb1c0421347fd31c9f87e31c9ac86978c09cc
+
+      - type: file
+        path: org.cockpit_project.CockpitClient.metainfo.xml
+finish-args:
```

Comment:
finish-args are usually above modules as a common practice

Reply:
OK.  I can move this.